src/tree/updater_quantile_hist.cc

```diff
@@ -1372,9 +1372,5 @@ XGBOOST_REGISTER_TREE_UPDATER(QuantileHistMaker, "grow_quantile_histmaker")
     []() {
       return new QuantileHistMaker();
     });
-
-template struct QuantileHistMaker::Builder<float>;
-template struct QuantileHistMaker::Builder<double>;
```

Comment:
JFYI: This was need on my compiler (g++ (GCC) 10.1.1). Did not follow though why.

Reply:
@boxdot  Thanks, we recently fixed it on master branch.  So the PR creates a conflict.